### ISSUE TITLE
Implement CHANGELOG and add missing 4.12 tag

### ISSUE THREAD

**Issue by chrisbloom7** [reidmorrison/rails_semantic_logger#185]
I got a dependabot upgrade PR to upgrade from 4.10 to 4.12. It would be helpful to be able to look at a changelog file to see what I should be on the look out for between minor version upgrades. Additionally, there is no 4.12 version tag so there's no way to compare from a previous version tag without looking at all the commits and finding the git sha for the 4.12 release. Could you commit to adding a changelog for future versions, and could you add a tag for 4.12 so future comparisons will be easier? (Both of these will also be used by dependabot to present changes between versions)

**Comment by adamzapasnik** [comment]
I agree, would be nice to have a changelog going forward @reidmorrison

**Comment by chrisbloom7** [comment]
@reidmorrison I see there are now tags for 4.12 and up. Thank you! I'd still love to see a CHANGELOG implemented

**Comment by reidmorrison** [comment]
v5 now has a complete changelog and the v4.12 tag is present.
